---
title: "Re: Question about counters"
description: ""
project: community
lastmod: 2014-07-01T01:40:32-07:00
sitemap:
  priority: 0.2
layout: mailinglistitem
mailinglist_id: "msg14430"
mailinglist_parent_id: "msg14416"
author_name: "Alex De la rosa"
project_section: "mailinglistitem"
sent_date: 2014-07-01T01:40:32-07:00
---


Hi there,

I found out why Riak was not starting when I enabled it... JAVA was not
installed!!... However, after installing Java and successfully started Riak
with search enabled, when trying to use "search" it crashes very badly,
this is the Java Version I have under Ubuntu Server 14.04 LTS:

java version "1.7.0\_60"
Java(TM) SE Runtime Environment (build 1.7.0\_60-b19)
Java HotSpot(TM) 64-Bit Server VM (build 24.60-b09, mixed mode)

This is my testing code using Python library:
-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----

import riak
client = riak.RiakClient(protocol='pbc',
nodes=[{'host':'127.0.0.1','http\_port':8098,'pb\_port':8087}])
bucket = client.bucket('accounts')
bucket.enable\_search()
key = bucket.new('alex', data={"username":"Alex","age":25,"sex":"male"},
content\_type='application/json')
key.store()
print bucket.search('sex=male')

# Code following example in:
http://basho.github.io/riak-python-client/query.html#fulltext-search

-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----
This is the error that Riak Search is giving to me:
-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----

Traceback (most recent call last):
 File "test.py", line 7, in 
 bucket.search('sex=male')
 File
"/usr/local/lib/python2.7/dist-packages/riak-2.0.3-py2.7.egg/riak/bucket.py",
line 446, in search
 File
"/usr/local/lib/python2.7/dist-packages/riak-2.0.3-py2.7.egg/riak/client/transport.py",
line 127, in wrapper
 File
"/usr/local/lib/python2.7/dist-packages/riak-2.0.3-py2.7.egg/riak/client/transport.py",
line 69, in \_with\_retries
 File
"/usr/local/lib/python2.7/dist-packages/riak-2.0.3-py2.7.egg/riak/client/transport.py",
line 125, in thunk
 File
"/usr/local/lib/python2.7/dist-packages/riak-2.0.3-py2.7.egg/riak/client/operations.py",
line 428, in fulltext\_search
 File
"/usr/local/lib/python2.7/dist-packages/riak-2.0.3-py2.7.egg/riak/transports/pbc/transport.py",
line 453, in search
 File
"/usr/local/lib/python2.7/dist-packages/riak-2.0.3-py2.7.egg/riak/transports/pbc/connection.py",
line 43, in \_request
 File
"/usr/local/lib/python2.7/dist-packages/riak-2.0.3-py2.7.egg/riak/transports/pbc/connection.py",
line 55, in \_recv\_msg
riak.RiakError: 'No index &lt;&lt;"accounts"&gt;&gt; found.'
-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----

Cheers,
Alex
\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
riak-users mailing list
riak-users@lists.basho.com
http://lists.basho.com/mailman/listinfo/riak-users\_lists.basho.com

